example/ROS_demo/face_detector_node.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+*  Copyright 2024 (C) Richard Kroesen <RichardKroesen>, Jeroen Veen <ducroq> & Victor Hogeweij <Hoog-V>
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+* This file is part of the Face_Detector.Cpp library
+*
+* Author:         Jeroen Veen <ducroq>
+*
+*/
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/image.hpp"
+#include "sensor_msgs/msg/region_of_interest.hpp"
+#include "cv_bridge/cv_bridge.hpp"
+#include <opencv2/opencv.hpp>
+#include "std_msgs/msg/int32.hpp"
+#include "geometry_msgs/msg/point_stamped.hpp"
+#include "std_srvs/srv/set_bool.hpp"
+#include <face_detection.hpp>
+
+class FaceDetectorNode : public rclcpp::Node
+{
+public:
+    FaceDetectorNode() : Node("face_detector_node")
+    {
+        declare_and_get_parameters();
+        det_.load_model(CFML_FACE_DETECTOR_CPU_MODEL_PATH);
```

Comment:
I see a hardcoded CPU Model, Isn't there a way to pass into the constructor parameters a option to select between TPU/CPU inference, or is it not supported at all?

Reply:
I do not know if TPU is supported. And cannot test it right. Maybe you can add this comment as a todo or issue or somethin to the project?